devices/tuya.js

```diff
@@ -3417,8 +3417,7 @@ module.exports = [
         ],
     },
     {
-        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_qoy0ekbd'},
-            {modelID: 'TS0601', manufacturerName: '_TZE200_znbl8dj5'}],
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_znbl8dj5'}],
```

Comment:
`_TZE200_qoy0ekbd` and `_TZE200_znbl8dj5` are already specified here, is this the same device? (CX-0726)

Reply:
tuya developer account, device PID, screenshot
https://github.com/13717033460/screenshots/blob/main/ZG-227ZL.png
https://github.com/13717033460/screenshots/blob/main/ZG-227z.png
https://github.com/13717033460/screenshots/blob/main/ZG-102zl.png